packages/v4/src/content/design-guidelines/components/tooltip/tooltip.md

```diff
@@ -2,51 +2,44 @@
 id: Tooltip
 section: components
 ---
-A **tooltip** provides short, clarifying descriptions of elements on a page. They are typically used to clarify the meaning of icons.
+A **tooltip** is in-app messaging used to identify elements on a page with short, clarifying text.
 
-## Usage
+For information on other forms of on-screen help, see [Popover](/components/popover/design-guidelines) and [Hint](/components/hint/design-guidelines). 
 
-<img src="./img/tooltips.png" alt="Example of tooltip in a table"  width="990"/>
+## Usage 
+Use tooltips to:
+- Provide labels for unlabeled icons.   
 
-Use tooltips to define new or unfamiliar UI elements that aren’t described directly in the user interface, or to get additional data from a data point or element in a chart or table.
+<img src="./img/kebab-tooltip.png" alt="kebab icon with a “More options” tooltip label" width="582"/>  
```

Comment:
For some reason this image looks tiny and the border on the right seems to get cut off? Like there's a broken grey line on the edge

Reply:
Replaced image